packages/uicore/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@harness/uicore",
-  "version": "3.148.7",
+  "version": "3.148.8",
```

Comment:
If you are not fixing any existing bug then minor version should be increased, not patch.

Reply:
Done